inabox/tests/integration_v2_test.go

```diff
@@ -138,59 +106,88 @@ var _ = Describe("Inabox v2 Integration", func() {
 		Expect(err).Error()
 		Expect(err.Error()).To(ContainSubstring(getSolidityFunctionSig("ABNNotInFuture(uint32)")))
 
-		// ensure that a verifier can be added two blocks in the future
+		// ensure that a verifier can be added two blocks in the future where activation_block_number = latestBlock + 2
 		tx, err := eigenDACertVerifierRouter.AddCertVerifier(deployerTransactorOpts, uint32(latestBlock)+2, gethcommon.HexToAddress("0x0"))
 		Expect(err).To(BeNil())
-
 		mineAnvilBlocks(1)
-		latestBlock += 1
 
 		// ensure that tx successfully executed
-		receipt, err := ethClient.TransactionReceipt(ctx, tx.Hash())
+		err = validateTxReceipt(ctx, tx.Hash())
 		Expect(err).To(BeNil())
-		Expect(receipt).To(Not(BeNil()))
-		Expect(receipt.Status).To(Equal(uint64(1)))
 
 		// ensure that new verifier can be read from the contract at the future rbn
-		verifier, err := eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock+1))
+		verifier, err := eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock+2))
 		Expect(err).To(BeNil())
 		Expect(verifier).To(Equal(gethcommon.HexToAddress("0x0")))
 
 		// and that old one still lives at the latest block number - 1
-		verifier, err = eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock-1))
+		verifier, err = eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock-2))
```

Comment:
`latestBlock-2`

This doesn't seem to match the comment `// and that old one still lives at the latest block number - 1`

Is there a reason that doing `latestBlock-1` in the code didn't work? It seems like it should...